WebView/app/src/main/assets/app.css

```diff
@@ -14,20 +14,34 @@
  * limitations under the License.
  */
 
-body {
-  background-color: white;
-}
+/* Dark Theme */
 @media (prefers-color-scheme: dark) {
-   body {
-     background: blue;
+  body {
+     background: #121212;
+  }
+   h1 {
+     color: white;
+   }
+   h2 {
+     color: white;
    }
 }
-h1 {
-  color: white;
-  text-align: center;
+
+/* Light Theme */
+@media (prefers-color-scheme: light) {
```

Comment:
What if there's no color-scheme set? Older WebView versions could have the "no-preference" scheme (and even older versions don't support the media query at all).

Reply:
This is what the app looks like on older versions of WebView. 
<img width="323" alt="Screen Shot 2020-07-20 at 9 12 54 AM" src="https://user-images.githubusercontent.com/50959941/87941530-4965b300-ca69-11ea-967c-c79891587fbf.png">

The system is set to dark mode here and so the sharing screen and drop down menu are dark. Would it make sense to override system preference in that case and have those be light theme if dark theme in WebView is not supported?